libraries/botbuilder-dialogs-adaptive-teams/src/actions/MessagingExtensionAttachmentLayoutResponseType.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @module botbuilder-dialogs-adaptive-teams
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Enum representing Messaging Extension Attachment Layout types.
+ */
+export enum MessagingExtensionAttachmentLayoutResponseType {
```

Comment:
Shouldn't this already exist in botframework-schema?

If this is duplicated in C# we need to redirect Microsoft.Bot.Builder.Dialogs.Adaptive.Teams.MessagingExtensionAttachmentLayoutResponseType to the existing enum in Microsoft.Bot.Schema.

Reply:
Ah, yeah. I see them [here](https://github.com/microsoft/botbuilder-js/blob/main/libraries/botframework-schema/src/teams/index.ts#L1486). Will adjust

Reply:
Thanks!